### DISCUSSION TITLE
https://blog.plumbiu.top/posts/blog/nextjs-theme

### DISCUSSION THREAD

**Discussion by giscus** [Plumbiu/blog#21] [Announcements]
# https://blog.plumbiu.top/posts/blog/nextjs-theme

在 CSR（客户端渲染）的情况下，主题切换往往不会出现问题，因为页面内容是 JS

https://blog.plumbiu.top/posts/blog/nextjs-theme

**Comment by Plumbiu** [comment]
测试一下 :smile:
